xpdacq/md_schema_v1.json

```diff
@@ -0,0 +1,153 @@
+{
+  "type": "object",
+  "properties":{
+    "bt_piLast":{
+      "type": "string",
+      "description": "Last name of PI"
+    },
+    "bt_wavelength":{
+      "type": "number",
+      "description": "x-ray wavelength"
+    },
+    "bt_safN":{
+      "type": "string",
+      "description": "Safety Approval Number of beamtime"
+    },
+    "bt_experimenters":{
+      "type": "array",
+      "description": "list of experimenter names in [first, last,...] format"
```

Comment:
See https://www.kalzumeus.com/2010/06/17/falsehoods-programmers-believe-about-names/

I suggest just storing as `[name1, name2, name3, ...]`, this schema should not be in the business of encoding the semantics of names (I go by 'Thomas A Caswell' or 'Thomas Caswell' or 'Thomas Alexander Caswell', all of which have different semantic than 'Stefan van der Walt' (lead skimage dev)).